packages/core/contracts/polygon/OracleBaseTunnel.sol

```diff
@@ -5,8 +5,7 @@ import "../oracle/interfaces/FinderInterface.sol";
 import "../oracle/implementation/Constants.sol";
 
 /**
- * @notice Enforces lifecyle of price requests for deriving contract. Ensures that price request data is not duplicated
```

Comment:
- In `OracleBaseTunnel` on [line 8](https://github.com/UMAprotocol/protocol/blob/a3bf46270787cbaae4ed2218f064b1217c153a50/packages/core/contracts/polygon/OracleBaseTunnel.sol#L8) "lifecyle" should be "lifecycle".